resources/ts/degree-program-overview/degree-program-overview.ts

```diff
@@ -0,0 +1,86 @@
+import { _x } from '@wordpress/i18n';
+
+import DegreeProgram, { DegreeProgramApiData } from './degree-program';
+import { LANGUAGE_SKILLS_INPUT } from '../filters/filters-handler';
+
+const DEGREE_PROGRAMS_SECTION_SELECTOR = '.c-degree-programs-search';
+const DEGREE_PROGRAMS_OVERVIEW_SELECTOR = '.c-degree-programs-collection';
+const SINGLE_PROGRAM_PREVIEW_SELECTOR = '.c-degree-program-preview';
+const NO_SEARCH_RESULT_SELECTOR = '.c-no-search-results';
+
+const degreeProgramsSection = document.querySelector(
+	DEGREE_PROGRAMS_SECTION_SELECTOR
+) as HTMLElement;
+const degreeProgramsOverview = degreeProgramsSection?.querySelector(
+	DEGREE_PROGRAMS_OVERVIEW_SELECTOR
+) as HTMLElement;
```

Comment:
It's hard to understand whether we are sure these elements exist. I suggest being unsure to prevent errors.
```suggestion
const degreeProgramsSection = document.querySelector< HTMLElement >(
	DEGREE_PROGRAMS_SECTION_SELECTOR
);
const degreeProgramsOverview =
	degreeProgramsSection?.querySelector< HTMLElement >(
		DEGREE_PROGRAMS_OVERVIEW_SELECTOR
	);
```
 So, we need to use optional chaining or check variables every time (like `toggleLanguageCertificateColumn`).